<launch>
  <arg
    name="lmm" />
  <param
    name="robot_description"
    textfile="$(find lmm_model)/urdf/lmm.urdf" />
  <node
    name="joint_state_publisher"
    pkg="joint_state_publisher"
    type="joint_state_publisher" />
    <rosparam
      param="source_list">
      ["lmm_joint_states"]
    </rosparam>
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="robot_state_publisher" />
  <node
    name="joy_node"
    pkg="joy"
    type="joy_node" />
  <node
    name="i2cpwm_board"
    pkg="i2cpwm_board"
    type="i2cpwm_board" />
  <node
    name="tip_tf_broadcaster"
    pkg="lmm_sc"
    type="tip_tf_broadcaster.py" />
  <node
    name="joy_incremental"
    pkg="lmm_sc"
    type="joy_incremental.py" />
  <node
    name="motor_controller"
    pkg="lmm_sc"
    type="motor_controller.py" />
  <node
    name="lmm_sc_master"
    pkg="lmm_sc"
    type="master.py" output="screen" />
</launch>
